src/main/java/com/intuit/graphql/orchestrator/stitching/XtextStitcher.java

```diff
@@ -284,16 +285,18 @@ private Map<String, GraphQLType> getAdditionalTypes(UnifiedXtextGraph unifiedXte
     XtextToGraphQLJavaVisitor localVisitor = XtextToGraphQLJavaVisitor.newBuilder().graphqlObjectTypes(visited).build();
 
     return unifiedXtextGraph.getTypes().values().stream()
+        .filter(type -> !type.getName().endsWith("ResolverArgument"))
         .filter(type -> !unifiedXtextGraph.isOperationType(type))
         .filter(type -> !(visited.containsKey(type.getName())))
-        .collect(Collectors.toMap(type -> type.getName(), type -> (GraphQLType) localVisitor.doSwitch(type)));
+        .collect(Collectors.toMap(TypeDefinition::getName, type -> (GraphQLType) localVisitor.doSwitch(type)));
   }
 
   private Set<GraphQLDirective> getAdditionalDirectives(UnifiedXtextGraph unifiedXtextGraph, Map<String, GraphQLDirective> visited) {
     XtextToGraphQLJavaVisitor localVisitor = XtextToGraphQLJavaVisitor.newBuilder().directiveDefinitions(visited)
         .build();
 
     return unifiedXtextGraph.getDirectives().stream()
+        .filter(directiveDefinition -> !directiveDefinition.getName().equals("resolver"))
```

Comment:
there may exist a constant for "resolver". If  not lets create one for "resolver" and "rename"